File: Sem1/ass11/gcd.c
#include<stdio.h>
int gcd(int, int);
int main()
{
	int n1,n2,res;
	printf("Enter two number:");
	scanf("%d%d",&n1,&n2);
	res=gcd(n1,n2);
	printf("GCD is %d",res);
}
int gcd(int a, int b)
{
	if(a==0)
		return b;
	if(b==0)
		return a;
	if(a>b)
		return gcd(a%b,b);
	else
		return gcd(a,b%a);
}

File: Sem3/a1/gcd.java
import java.util.Scanner;
public class gcd {
    public static void main(String[] args) {
        int i,m,n,gcd=1;
        Scanner sc= new Scanner(System.in);
        System.out.print("Enter two numbers: ");
        m= sc.nextInt();
        n= sc.nextInt();
        for(i=1;i<=m&&i<=n;i++){
            if(m%i==0 && n%i==0){
                gcd=i;
            }
        }
        System.out.println(gcd+" is the gcd of "+m+" and "+n);
    }
}
